auto_backup/model/backup_scheduler.py

```diff
@@ -0,0 +1,376 @@
+            "will be removed after 5 days."
+        ), required=True
+    )
+    sftpwrite = fields.Boolean(
+        string='Write to external server with sftp',
+        help=(
+            "If you check this option you can specify the details "
+            "needed to write to a remote server with SFTP."
+        )
+    )
+    sftppath = fields.Char(
+        string='Path external server',
+        help=(
+            "The location to the folder where the dumps should be "
+            "written to. For example /odoo/backups/.\nFiles will then"
+            " be written to /odoo/backups/ on your remote server."
+        )
+    )
+    sftpip = fields.Char(
+        string='IP Address SFTP Server',
+        help=(
+            "The IP address from your remote"
+            " server. For example 192.168.0.1"
+        )
+    )
+    sftpport = fields.Integer(
+        string="SFTP Port",
+        default=22,
+        help="The port on the FTP server that accepts SSH/SFTP calls."
+    )
+    sftpusername = fields.Char(
+        string='Username SFTP Server',
+        help=(
+            "The username where the SFTP connection "
+            "should be made with. This is the user on the external server."
+        )
+    )
+    sftppassword = fields.Char(
+        string='Password User SFTP Server',
+        help=(
+            "The password from the user where the SFTP connection "
+            "should be made with. This is the password from the user"
+            " on the external server."
+        )
+    )
+    daystokeepsftp = fields.Integer(
+        string='Remove SFTP after x days',
+        default=30,
+        help=(
+            "Choose after how many days the backup should be deleted "
+            "from the FTP server. For example:\nIf you fill in 5 the "
+            "backups will be removed after 5 days from the FTP server."
+        )
+    )
+    sendmailsftpfail = fields.Boolean(
+        string='Auto. E-mail on backup fail',
+        help=(
+            "If you check this option you can choose to automaticly"
+            " get e-mailed when the backup to the external server failed."
+        )
+    )
+    emailtonotify = fields.Char(
+        string='E-mail to notify',
+        help=(
+            "Fill in the e-mail where you want to be"
+            " notified that the backup failed on the FTP."
+        )
+    )
+    lasterrorlog = fields.Text(
+        string='E-mail to notify',
+        help=(
+            "Fill in the e-mail where you want to be"
+            " notified that the backup failed on the FTP."
+        )
+    )
+
+    @api.multi
+    def _check_db_exist(self):
+        for rec in self:
+            db_list = db.exp_list(True)
+            if rec.name in db_list:
+                return True
+            return False
+
+    _constraints = [
+        (
+            _check_db_exist,
+            _('Error ,No such database exists'), ['name']
+        )
+    ]
```

Comment:
New-api constraints should use the `@constrains` decorator unless they are SQL constraints. Could you please update this?